eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -1,8 +1,9 @@
+# Template to evaluate common paths in different pipelines.
 parameters:
   extraSubsets: ''
 
 jobs:
-- template: /eng/pipelines/common/checkout-job.yml
+- template: /eng/pipelines/common/evaluate-paths-job.yml
```

Comment:
presumably that template would only be imported if dependOnEvaluatePaths is set but you mentioned in the stand-up that this isn't easily possible?

Reply:
Not really. This template needs to be imported always here, as that is the point of `evaluate-default-paths.yml`. However, this template is only referenced in pipelines that need it (`runtime` and `runtime-staging`) and also, this job is only expanded if `Build.Reason==PullRequest`

Reply:
Sorry, I meant the whole evaluate-default-paths.yml file.

Reply:
See my comment here: https://github.com/dotnet/runtime/pull/46107#discussion_r544597671